kubetest/extract_test.go

```diff
@@ -137,6 +137,16 @@ func TestExtractStrategies(t *testing.T) {
 			"https://storage.googleapis.com/gke-release-staging/kubernetes/release",
 			"v1.2.3+abcde",
 		},
+		{
+			"ci/gke-latest-1.13.0",
+			"https://storage.googleapis.com/gke-release-staging/kubernetes/release",
+			"v1.2.3+abcde",
+		},
+		{
+			"ci/gke-latest-1.13.0-gke",
```

Comment:
I obviously didn't write the most useful test

Reply:
lol